wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,297 @@
+
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) introduces an alternative wallet detection mechanism to the `window.ethereum` provider, and enables dapps to support [wallet interoperability](../concepts/wallet-interoperability.md).
+
+For a full end-to-end tutorial that can be used in production, see the
+[Create a simple React dapp](../tutorials/react-dapp-local-state.md) tutorial.
+:::
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) version 20
+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 9+
+
+- A text editor of your choice, such as [VS Code](https://code.visualstudio.com/).
+
+- [MetaMask](https://metamask.io/) installed in the browser of your choice on your development machine.
+
+## Steps
+
+### 1. Set up a new project
+
+Create a new project using [Vite](https://vitejs.dev/guide/):
+
+```bash
+npm create vite@latest simple-dapp -- --template vanilla
+```
+
+Change directories:
+
+```bash
+cd simple-dapp 
+```
+
+Install the dependencies listed in the project's `package.json`.
+
+```bash
+npm install    
+```
+
+Run the development script to start a local development server:
+
+```bash
+npm run dev
+```
+
+### 2. Detect MetaMask
+
+:::caution
+
+The `@metamask/detect-provider` module is deprecated, and is only used here for educational purposes. In production environments, we recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md).
+
+:::
+
+Install the `@metamask/detect-provider` module in your project directory:
+
+```bash
+npm i @metamask/detect-provider
+```
+
+Create a `src` directory, change into it, and create a new file `detect.js`:
+
+```bash
+mkdir src
+cd src
+touch detect.js
+```
+
+Your project structure should look similar to the following:
+
+```text
+simple-dapp/
+├─ src/
+│  ├─ detect.js
+├─ dist/
+│  ├─ index.html
+```
+
+In a text editor, add the following code to `src/detect.js` to detect the MetaMask provider using `@metamask/detect-provider`:
+
+```js title="detect.js"
+import detectEthereumProvider from "@metamask/detect-provider";
+
+async function setup() {
+  const provider = await detectEthereumProvider();
+
+  if (provider && provider === window.ethereum) {
+    console.log("MetaMask is available!");
+    startApp(provider); // Initialize your dapp with MetaMask.
+  } else {
+    console.log("Please install MetaMask!");
+  }
+}
+
+function startApp(provider) {
+  if (provider !== window.ethereum) {
+    console.error("Do you have multiple wallets installed?");
+  }
+}
+
+window.addEventListener("load", setup);
+```
+
+### 3. Serve the project
+
+To test and verify your project works with MetaMask, serve your project through a local server. 
+Navigate to the simple-app project directory.
+Run the `npm run dev` command to test your application locally. 
+Open your browser and go to the provided local server URL.
+
```

Comment:
```suggestion
```

Nothing has actually changed at this point, since we haven't updated `index.html`. Let's remove this step and just add a screenshot at the final stage of the tutorial.